### PR TITLE
[MODINV-1080] API version update

### PR DESCRIPTION
**Pull request by RomanChernetskyi**
### Purpose
The Spitfire team is going to introduce the changes that will lead to the API major version update in this PR [mod-inventory-storage#1084](https://github.com/folio-org/mod-inventory-storage/pull/1084)

### Approach
Update provided interfaces:
- inventory from '13.3' to '14.0'
- inventory-batch from '2.0' to '3.0'

Update required interfaces:
- instance-storage from '10.3' to '11.0'
- instance-storage-batch from '2.0' to '3.0'

### CODE REVIEW

**Review by KaterynaSenchenko — APPROVED** [review]

File: ramls/instance.json [hunk 1]
```diff
@@ -287,21 +287,6 @@
       },
       "uniqueItems": true
     },
-    "publicationPeriod": {
```
Comment (RuslanLavrov) [inline ramls/instance.json:290]: please, update Instance class as well to remove manipulations with "publicationPeriod" field there 
https://github.com/folio-org/mod-inventory/blob/master/src/main/java/org/folio/inventory/domain/instances/Instance.java#L77

Comment (RomanChernetskyi) [reply ramls/instance.json:290]: Hi, @RuslanLavrov, Removed, please review, thanks

Comment (RuslanLavrov) [reply ramls/instance.json:290]: Hi @RomanChernetskyi, thanks!

File: descriptors/ModuleDescriptor-template.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "inventory",
-      "version": "13.3",
+      "version": "14.0",
```
Comment (RuslanLavrov) [inline descriptors/ModuleDescriptor-template.json:7]: it looks like we should also increase version for "inventory-batch" interface

Comment (RomanChernetskyi) [reply descriptors/ModuleDescriptor-template.json:7]: @RuslanLavrov done, thank you

Comment (RuslanLavrov) [reply descriptors/ModuleDescriptor-template.json:7]: thank you

**Pull request merged**